UIKitDZ/UIKitDZ/FourViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  FourViewController.swift
+//  UIKitDZ
+//
+//  Created by Maxim on 15.10.2022.
+//
+
+import UIKit
+/// Контроллер Stack View
+final class FourViewController: UIViewController {
+    
+    // MARK: - Visual Components
+    private lazy var redView: UIView = {
```

Comment:
```suggestion
    private let redView: UIView = {
```
везде где позволяет компилятор поменять var  на  let